client/src/components/DataFiles/DataFilesModals/DataFilesProjectEditDescriptionModal.jsx

```diff
@@ -62,18 +70,21 @@ const DataFilesProjectEditDescriptionModal = () => {
   const validationSchema = Yup.object().shape({
     title: Yup.string()
       .min(3, 'Title must be at least 3 characters')
-      .max(150, 'Title must be at most 150 characters')
+      .max(maxTitleLength, `Title must be at most ${maxTitleLength} characters`)
       .required('Please enter a title.'),
-    description: Yup.string().max(
-      800,
-      'Description must be at most 800 characters'
-    ),
+    description: Yup.string()
```

Comment:
Can we add a regex pattern validator here that ensures keywords are comma separated strings?

Reply:
Great catch, added!